File: CayGiaPhaAPI/CayGiaPhaAPI/Class/Person.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace CayGiaPhaAPI.Class
{
    public class Person
    {
        [Key]

        public int personId { get; set; }
        public string personName { get; set; }
        public string personGender { get; set; }
        public string personImg { get; set; }

        public int? IdNodeCha { get; set; }
    }
}

File: CayGiaPha/src/app/shared/models/Person.ts
export class Person {
  personId: number;
  personName: string;
  personGender: string;
  personImg: string;
  IdNodeCha: Number;

  constructor() {
    this.personId = 0;
    this.personName = "";
    this.personGender = "";
    this.personImg = "";
    this.IdNodeCha = 0;
  }

  copy(person: Person) {
    this.personId = person.personId;
    this.personName = person.personName;
    this.personGender = person.personGender;
    this.personImg = person.personImg;
    this.IdNodeCha = person.IdNodeCha;
  }
}
